src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx

```diff
@@ -76,18 +76,20 @@ Software built at New Relic goes through these five phases.
 </table>
 
 ## Requirements
-Each project team is assigned a security engineer charged with ensuring the security of New Relic products. In the requirements building phase, the security engineer performs a risk assessment and then adds security requirements for the project. We add privacy and compliance experts to the project teams as needed.
+Each project team for our generally available New Relic One platform is assigned a security engineer charged with reviewing and advision on the security of New Relic products. In the requirements building phase the security engineer performs a risk assessment and then adds security requirements for the project. Privacy and compliance experts are added to the project teams as needed.
```

Comment:
@morrisco, could you clarify "reviewing and advision on"? Is this a typo?

Reply:
This came from the collaboration with legal, I can sync up with them again. Is the issue clarity?

Reply:
That was a typo. Fix commited in the PR